applications/pelion_client/configuration/common/update_default_resources.c

```diff
@@ -0,0 +1,39 @@
+// ----------------------------------------------------------------------------
```

Comment:
Why doesn't checkpatch complain about `//` comments is a mystery to me.

Reply:
it's not perfect... anyway this file is a stub that is substituted by real file generated from tools. I missed it and will clean it up.

Reply:
fixed